backend/scripts/airtable/dumpProposals.js

```diff
@@ -0,0 +1,146 @@
+require("dotenv").config();
+const airtable = require("airtable");
+const mongoose = require("mongoose");
+const Project = require("../../models/Project");
+const Proposal = require("../../models/Proposal");
+const fetch = require("node-fetch");
+const ethers = require("ethers");
+const showdown = require("showdown");
+
+airtable.configure({
+  apiKey: process.env.AIRTABLE_API_KEY,
+});
+const grantCategoryJson = require("../../utils/types/grant_category.json");
+const earmarkJson = require("../../utils/types/earmark.json");
+let db = mongoose.connection;
+const base = airtable.base(process.env.AIRTABLE_BASE_ID);
+
+// connect to localhost
+mongoose.connect(
+  `mongodb://${process.env.MONGODB_URI}/${process.env.MONGODB_NAME}`
+);
+
+db.on("connected", () => {
+  console.log("Mongoose connected");
+  dumpData();
+});
+
+// on error
+db.on("error", (err) => {
+  console.log("Mongoose connection error: " + err);
+});
+
+// on disconnected
+db.on("disconnected", () => {
+  console.log("Mongoose disconnected");
+});
+
+async function dumpData() {
```

Comment:
I have reviewed the implementation at a high level. I have not double checked every assigned to see if it makes sense. I believe this looks to be generally correct.